dev-libs/OpenNI/OpenNI-1.5.7.10-r4.ebuild

```
@@ -53,7 +61,8 @@ src_prepare() {
 		echo "" > ${i}
 	done
 
-	find . -type f -print0 | xargs -0 sed -i "s:\".*/SamplesConfig.xml:\"${EPREFIX}/usr/share/${PN}/SamplesConfig.xml:" || die
+	find . -type f -print0 \
```

I think the `\` is unnecessary

```suggestion
	find . -type f -print0
```

---

it is for `ExcessiveLineLength	excessive line length (over 120 characters) on line: 56` in https://qa-reports.gentoo.org/output/gentoo-ci/output.verbose.html;pkg=dev-libs:OpenNI